tools/tpm2_unseal.c

```diff
@@ -111,13 +117,17 @@ static bool init(int argc, char *argv[], tpm_unseal_ctx *ctx) {
             UINT8 H : 1;
             UINT8 c : 1;
             UINT8 P : 1;
+            UINT8 L : 1;
+            UINT8 F : 1;
         };
         UINT8 all;
     } flags = { .all = 0 };
 
     int opt;
     bool hexPasswd = false;
     char *contextItemFile = NULL;
+    TPML_PCR_SELECTION pcr_selections;
+    TPM2B_DIGEST pcr_hashes = TPM2B_TYPE_INIT(TPM2B_DIGEST, buffer);
     while ((opt = getopt_long(argc, argv, optstring, long_options, NULL)) != -1) {
```

Comment:
I'll submit a patch with optind = 0 here. Surprised we haven't hit any weird parsing errors.